server/graphql/v2/interface/Account.js

```diff
@@ -595,6 +597,26 @@ const accountFieldsDefinition = () => ({
       return pick(policies, PUBLIC_POLICIES);
     },
   },
+  activitySubscriptions: {
+    type: new GraphQLNonNull(new GraphQLList(ActivitySubscriptions)),
+    description: 'List of activities that the logged-in user is subscribed for this collective',
+    args: {
+      channel: {
+        type: ActivityChannel,
+      },
+    },
+    async resolve(collective, args, req) {
+      if (!req.remoteUser) {
```

Comment:
scope check

Reply:
Also I'd be tempted to return `null` if no `remoteUser` or doesn't have the right scope, to differentiate with cases where the list is really empty. No blocker.